assignments/Assign11CDogFactsAPI.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Dog Breeds</title>
+</head>
+<body>
+    <header>
+        <h2>List of dog breeds</h2>
+        <input type="button" id="listBreeds">List Breeds</input>
+    </header>
+
+    <div id="breed"></div>
+
+    <script>
+            function main() {
+
+                const getBreeds = () => {
+
+                     //create instance of xml object
+                    const xhttp = new XMLHttpRequest();
+
+                    //specify method and url with input values
+                    const reqUrl = `https://dogapi.dog/api/v2/breeds`;
+
+                    xhttp.open("GET", reqUrl, true);
+
+                    // load data from server and give error message if needed
+                    xhttp.onload = () => {
+                        if (xhttp.status >= 200 && xhttp.status < 400) {
+                            const resData = JSON.parse(xhttp.responseText);
+                            renderHTML(resData);
+                        } else {
+                            return "Server connection made but there is an error.";
```

Comment:
Possibly something I didn't catch on your first writing of this code, but this (any most any event handler) should not return a value - nothing will consume that value and it goes nowhere.